packages/client/src/models/mentor.js

```diff
@@ -76,12 +156,13 @@ export default class Mentor {
 
     /**
      * Publishes the current mentor instance to firebase
-     * @param {firebase.auth.UserCredential} the firebase user object
+     * @param {firebase.auth.UserCredential} user object from firebase
      *
      * @return {Promise<void>} a promise indicating successful creation.
      */
     create(user) {
-        this.validate();
+        this.validate().catch(reason => console.log(reason));
```

Comment:
I think it would be cool to just throw the error. This way the component that caused it can display it to the user.

Reply:
So remove the catch?

Reply:
I think that will do the trick.